docs/rfcs/0003-initial.md

```diff
@@ -0,0 +1,280 @@
+# x/database/sql for Deno, Version 0.1
+
+**Date:** February 28, 2021
+
+**Status:**
+[Your comments are requested!](https://github.com/jeremyBanks/database/pull/3)
+
+## Background
+
+Deno currently has several great driver libraries for different database
+engines. However, each one provides a slightly different interface, with a
+slightly different set of functionality, creating friction for users who want to
+write code that works across database engines. This is similar to the position
+that Go was in that led to the creation of their `database/sql` library. Deno
+isn't exactly the same as Go, but they have a lot in common. With some changes,
+we can take Go's API design and their
+["Goals of the `sql` and `sql/driver`
+packages"](https://golang.org/src/database/sql/doc.txt) document (below) as a
+starting point for our own library:
+
+> - Provide a generic database API for a variety of SQL or SQL-like databases.
+  > There currently exist Go libraries for SQLite, MySQL, and Postgres, but all
+  > with a very different feel, and often a non-Go-like feel.
```

Comment:
```suggestion
  > There currently exist Deno libraries for SQLite, MySQL, and Postgres, but all
  > with a very different feel, and often a non-Deno-like feel.
```

Reply:
I think we should keep the `Go`, since it's a direct quote 😅

Reply:
As dyedgreen says, I'm going to leave these because it's an exact quote, but if that's not clear I'm open to suggestions for improving the framing.